src/oci-umount.c

```diff
@@ -369,21 +432,34 @@ static int prestart(const char *rootfs,
 	}
 
 	while ((read = getline(&line, &len, fp)) != -1) {
+		bool submounts_only = false;
```

Comment:
You should grab the comments part of my patch to allow comments to be added to /etc/oci-umount.conf file.   We need an explanation of this behaviour in the config file.